ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -319,6 +319,12 @@
     reason="Test will not run on IBM cloud",
 )
 
+skipif_ibm_cloud_managed = pytest.mark.skipif(
+    config.ENV_DATA["deployment_type"].lower() != "managed"
```

Comment:
Could you please explain, do you want to skip the test for managed or not managed IBM cloud deployment? The name suggests you want to skip it for managed deployment but the condition is the other way around. Am I missing something?

Reply:
@ebondare you are completely right! Thank you for catching this error!